components/dashboard/src/start/CreateWorkspace.tsx

```diff
@@ -385,20 +386,21 @@ function SpendingLimitReachedModal(p: { hints: any }) {
     return (
         <Modal visible={true} closeable={false} onClose={() => {}}>
             <h3 className="flex">
-                <span className="flex-grow">Spending Limit Reached</span>
+                <span className="flex-grow">Usage Limit Reached</span>
             </h3>
             <div className="border-t border-b border-gray-200 dark:border-gray-800 mt-4 -mx-6 px-6 py-2">
-                <p className="mt-1 mb-2 text-base dark:text-gray-400">Please increase the spending limit and retry.</p>
+                <Alert type="error" className="app-container rounded-md">
+                    You have reached the <strong>spending limit</strong> of your billing account.
+                </Alert>
+                <p className="mt-1 mb-2 text-base dark:text-gray-400">
+                    Contant a team owner of <strong>{attributedTeam}</strong> to increase the spending limit, or change
+                    your <a href="/billing">billing settings</a>.
+                </p>
             </div>
             <div className="flex justify-end mt-6 space-x-2">
-                <a href={gitpodHostUrl.with({ pathname: "billing" }).toString()}>
-                    <button>Billing Settings</button>
+                <a href={gitpodHostUrl.asDashboard().toString()}>
```

Comment:
Nit: I'm not sure why we're using the over-engineered `gitpodHostUrl.with({ pathname: "billing" })` thing when we can just link to `/billing` like a normal web app 😅 

Also, using `<Link>` instead of `<a>` when possible makes link clicks much faster (single-page app navigation, as opposed to full reload of the app).

```suggestion
                <Link to="/">
```

Reply:
Me neither! I thought it was about some XSS safety mechanism. Just copied what was already there! 🤷

Reply:
@easyCZ Do you have any insights or objections on replacing the links with the `Link` since you added there?

Reply:
Opened https://github.com/gitpod-io/gitpod/issues/12233 to keep track of this discussion. Resolving.